src/scattering.F90

```diff
@@ -40,52 +41,68 @@ subroutine nu_scatter_elastic_e_total(neutrino_energy,type,lepton,eos_variables,
      stop "nu_scatter_elastic_e_total: lepton number weird...."
   endif
 
-  if (type.eq.1) then
-     crosssection = 3.0d0/8.0d0*sigma0/m_e**2*neutrino_energy* &
-          (eos_variables(tempindex)+eos_variables(mueindex)/4.0d0)* &
-          ((c_v_prime+c_a_prime)**2+(c_v_prime-c_a_prime)**2/3.0d0)
-  else
-     stop "no transport crosssection?"
+  !ONLY TRANSPORT OPACITY IMPLEMENTED
+  !if (transport.eq.1) then
+  crosssection = 3.0d0/8.0d0*sigma0/m_e**2*neutrino_energy* &
+       (eos_variables(tempindex)+eos_variables(mueindex)/4.0d0)* &
+       ((c_v_prime+c_a_prime)**2+(c_v_prime-c_a_prime)**2/3.0d0)
+  delta = 0
+  !else
+  !   stop "no transport crosssection?"
+  !endif
+
+  ! check output
```

Comment:
Can we do the checking either with a DEBUG compiler flag, or outside once it is all calculated.  I like to think we are trying to keep things running fast, doing an if for each and every calculation of delta might get expensive.

Reply:
Done.